docs/dev/developer-guides/contracts/contracts.md

```diff
@@ -31,7 +46,23 @@ Compilers are no longer released as Docker images and its usage is no longer rec
 
 :::
 
-Check out the documentation in the links below:
+**Learn more about how to install and configure these plugins in the links below:**
 
 - [hardhat-zksync-solc documentation](../../../api/hardhat/plugins.md#hardhat-zksync-solc)
 - [hardhat-zksync-vyper documentation](../../../api/hardhat/plugins.md#hardhat-zksync-vyper)
+
+## EVM compatibility
+
+Almost every smart contract written for EVM will be supported by zkSync 2.0 and will hold all key security invariants so that no additional security re-auditing will be required in most cases. A notable exception is the contracts that use the following EVM opcodes:
+
+- `SELFDESTRUCT`: it’s considered harmful and was deprecated in [EIP-6049](https://eips.ethereum.org/EIPS/eip-6049).
+- `EXTCODECOPY` - we've skip it for now because zkEVM opcodes are not identical to EVM ones anyway but it can be implemented if needed.
+- `CALLCODE` - it was deprecated on Ethereum in favor of `DELEGATECALL`.
```

Comment:
Referencing where it was stated would be nice.
Link - https://eips.ethereum.org/EIPS/eip-2488

Reply:
solved